PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py

```diff
@@ -48,7 +49,7 @@
 #add your new path here!!
 allMetFilterPaths=['HBHENoiseFilter','HBHENoiseIsoFilter','CSCTightHaloFilter','CSCTightHaloTrkMuUnvetoFilter','CSCTightHalo2015Filter','globalTightHalo2016Filter','globalSuperTightHalo2016Filter','HcalStripHaloFilter','hcalLaserEventFilter','EcalDeadCellTriggerPrimitiveFilter','EcalDeadCellBoundaryEnergyFilter','ecalBadCalibFilter','goodVertices','eeBadScFilter',
                    'ecalLaserCorrFilter','trkPOGFilters','chargedHadronTrackResolutionFilter','muonBadTrackFilter',
-                   'BadChargedCandidateFilter','BadPFMuonFilter','BadChargedCandidateSummer16Filter','BadPFMuonSummer16Filter',
+                   'BadChargedCandidateFilter','BadPFMuonFilter','BadPFMuonDzFilter','BadChargedCandidateSummer16Filter','BadPFMuonSummer16Filter',
```

Comment:
another difference wrt master
https://github.com/cms-sw/cmssw/blob/a37e1d9101e88827f6960623c8549894f3601cd4/PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py#L52

Similarly, to the above, it's probably better here in a sense of self-consistency of the style (no spaces), but then the master should be updated first.

Reply:
made consistent with master